### PR TITLE
chore: add volunteer aksh-kr

### PR DESCRIPTION
**Pull request by aksh-kr**

### CODE REVIEW

File: data/users/aksh-kr.json [hunk 1]
```diff
@@ -0,0 +1,14 @@
+{
+  "name": "Akash Kumar",
+  "bio": "Software Developer",
+  "image": "https://avatars.githubusercontent.com/u/53661628?v=4",
+  "categories": [
+    {
+      "slug": "programming | software-development",
```
Comment (codewithvoid) [inline data/users/aksh-kr.json:7]: just programming is fine
```json
"slug": "programming"
```

File: data/users/aksh-kr.json [hunk 2]
```diff
@@ -0,0 +1,14 @@
...
+      "topics": ["Java", "Data structures", "Algorithms"]
```
Comment (codewithvoid) [inline data/users/aksh-kr.json:8]: You can just mention DSA in topics
```json
  "topics": ["Java", "DSA"]
```

**Review by codewithvoid — CHANGES_REQUESTED** [review]

**Comment by codewithvoid** [comment]
Thanks @aksh-kr for volunteering with AskDevs. Added some comments on your change.

File: data/users/aksh-kr.json (hunk 2 above)
Comment (aksh-kr) [reply data/users/aksh-kr.json]: updated the JSON file as per the code review comments

**Review by codewithvoid — APPROVED** [review]

**Comment by codewithvoid** [comment]
Thanks @aksh-kr. Welcome!

**Comment by aksh-kr** [comment]
Thank you @codewithvoid!

**Pull request merged**